### ISSUE TITLE
Investigate using internal jira authentication

### ISSUE THREAD

**Issue by fossf** [Terracotta-OSS/terracotta.org-site#11]
rather than sign up via website. Must also keep existing users working.

**Comment by fossf** [comment]
check on active users -

**Comment by fossf** [comment]
does having ehcache2 on github make sense? for issues

**Comment by ljacomet** [comment]
I do not think so - we pretty much want Ehcache 2.x to become the legacy line sooner than later. And thus having GitHub focus on 3.x seems better for that.
